index.html

```diff
@@ -53,9 +53,11 @@ <h1 id="myName">
             </ul>
         </div>
       </section>
+      
       <section class="works">
         <div class="recent-works">
-          <h2 id="worksTitle">My Recent Works</h2>  
+          <h2 id="worksTitle">My Recent Works</h2> 
+
```

Comment:
- [ ] [OPTIONAL] According to the requirements of the project you must follow the[ figam design guidelines](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=48-27&t=V8Y95Xw5Xt48Kl9N-0) including the typography so I would suggest you to use font-family `Crete Round` for your heading `My Recent Works`.

Reply:
There are style lint errors that need a generic font how do you come about that

Reply:
Hi @Obimbo07,   The `font-family` property can hold several font names as a "fallback" system. If the browser does not support the first font, it tries the next font. the first font in the font-family indicates the font priority. When we use some fonts other than the generic one we always include a generic font in the list of font families so that if non of the fonts listed in the font family are supported by the browser it should fall back to the generic one. You can use the name of a generic-family (serif, sans-serif, ...) etc along with the font of your choice. For example:

```css

font-family: 'Inter', sans-serif;

```